src/Core/Form/IdentifiableObject/CommandBuilder/Product/SEOCommandsBuilder.php

```diff
@@ -57,7 +57,7 @@ public function buildCommands(ProductId $productId, array $formData): array
         }
 
         if (isset($redirectionData['type'])) {
-            $targetId = (int) ($redirectionData['target'] ?? 0);
+            $targetId = !empty($redirectionData['target']) ? (int) reset($redirectionData['target']) : 0;
```

Comment:
So, target is an array of ids? :thinking:

Reply:
Yes the form data format has slightly changed because it now relies on a generic `EntitySearchInputType` which is based on a `CollectionType` which can be used to assign multiple entities (even if this special case the limit is 1)

For now, I kept the array format, but I was hesitating to handle some data transformer on this form type so that the special case with limit == 1 could handle the data directly instead of an array (it would be a simple transform from singular value to "arrayfied" value, back and forth). But I didn't want to make this PR more complex, maybe this could be an improvement added in another PR?

Reply:
The data transformer should be the solution. Having word `target` for an array is a little bit weird. Otherwise, maybe the target can accept int|array|string  :thinking:

Reply:
I just added the transformer for this part, it's much better